lib/Distribution/Opensuse/AgamaDevel.pm

```diff
@@ -31,12 +32,18 @@ sub get_grub_menu_agama {
     });
 }
 
-sub get_grub_menu_installed_system {
+sub get_grub_menu_tumbleweed_installed_system {
     return Yam::Agama::Pom::GrubMenuTumbleweedPage->new({
             grub_menu_base => Yam::Agama::Pom::GrubMenuBasePage->new()
     });
 }
 
+sub get_grub_menu_leap_installed_system {
```

Comment:
you are adding this in the wrong distribution file, you don't need to rename the existing one.